tests/e2e/e2e_test.go

```diff
@@ -62,12 +62,8 @@ func getTestEnv() {
 
 	installCTK = getEnvVarOrDefault("E2E_INSTALL_CTK", false)
 
-	if installCTK {
-		imageName = getRequiredEnvvar[string]("E2E_IMAGE_NAME")
-
-		imageTag = getRequiredEnvvar[string]("E2E_IMAGE_TAG")
-
-	}
+	imageName = getRequiredEnvvar[string]("E2E_IMAGE_NAME")
+	imageTag = getRequiredEnvvar[string]("E2E_IMAGE_TAG")
```

Comment:
Is there a reason we removed the conditional?

Reply:
Yes, regardless of if we want to install or not the toolkit on the host, I want to be able to get these 2 variables.

Reply:
Why? What do we need the image for if we're not installing the toolkit?

Reply:
for the test cases added in this PR, we need to pass the image into the runner container, so we can get the artifacts to install the toolkit and the `nvidia-container-cli`